src/ui/components/TransferGrain.js

```diff
@@ -11,7 +11,7 @@ import {
 } from "../../ledger/grain";
 import {type Identity} from "../../ledger/identity";
 import {Ledger, type Account} from "../../ledger/ledger";
-import AccountDropdown from "./AccountSelector";
+import AccountDropdown from "./AccountDropdown";
```

Comment:
Looks like we added AccountDropdown without removing AccountSelector. Is there a reason we need both?

Reply:
It is used on other pages still. But I could make sure everything works with those pages too. Just figured that would lead to styling pages on other pages and this branch quickly getting out of scope.

Reply:
Yeah, let's update in both places please. I think the PR will still be small & easy to review, and that way we guarantee we aren't adding debt by having two components that do nearly the same thing. It also "validates" the new component a bit by showing that we can re-use it.

Reply:
Okay, I am using this in both places now. This did change how the SpecialGrainDistribution page looks a bit, but did not affect usability, and there will be styling updates there as well in the future. I didn't want to include css changes in this commit.